### PR TITLE
Use MultiPoint over Points collection in MapFragment

### PR DESCRIPTION
**Pull request by dellisd**
… MapFragment

Moderate performance gain.

### CODE REVIEW

File: app/src/main/java/ca/llamabagel/transpo/ui/map/MapFragment.kt [hunk 1]
```diff
@@ -17,6 +17,7 @@ import ca.llamabagel.transpo.R
 import ca.llamabagel.transpo.di.injector
 import com.mapbox.geojson.Feature
 import com.mapbox.geojson.FeatureCollection
```
Comment (GustavoSanMartin) [inline app/src/main/java/ca/llamabagel/transpo/ui/map/MapFragment.kt]: unused import

**Comment by dellisd** [comment]
It'll still be in the extended description. Not sure how to change the message..

**Comment by GustavoSanMartin** [comment]
tbh just
```
git reset HEAD^
git commit --amend
```
then edit the title

File: app/src/main/java/ca/llamabagel/transpo/ui/map/MapFragment.kt [hunk 2]
```diff
@@ -5,28 +5,24 @@
...
 import com.mapbox.geojson.Point
 import com.mapbox.mapboxsdk.Mapbox
 import com.mapbox.mapboxsdk.maps.MapView
 import com.mapbox.mapboxsdk.maps.MapboxMap
 import com.mapbox.mapboxsdk.maps.Style
-import com.mapbox.mapboxsdk.style.expressions.Expression.interpolate
-import com.mapbox.mapboxsdk.style.expressions.Expression.exponential
-import com.mapbox.mapboxsdk.style.expressions.Expression.zoom
-import com.mapbox.mapboxsdk.style.expressions.Expression.stop
+import com.mapbox.mapboxsdk.style.expressions.Expression.*
```
Comment (GustavoSanMartin) [inline app/src/main/java/ca/llamabagel/transpo/ui/map/MapFragment.kt]: I thought ktlint didn't allow wildcards?

**Pull request closed without merging**